lib/matplotlib/tests/test_figure.py

```diff
@@ -66,6 +66,31 @@ def test_align_labels():
     fig.align_labels()
 
 
+@image_comparison(['figure_align_titles'], extensions=['png', 'svg'],
+                  tol=0 if platform.machine() == 'x86_64' else 0.01,
+                  style='mpl20')
+def test_align_titles():
+    fig = plt.figure(layout='tight')
```

Comment:
Please also test layout='constrained'

Reply:
Would duplicating the test for both layouts be the best way to implement this? (i.e. ```test_align_titles_constrained()``` and ```test_align_titles_tight()```) Or is there a way to test both layouts in the same image comparison? Thanks!

Reply:
Yes you will need to duplicate the test.  pytest parametrization might be the cleanest way.
https://docs.pytest.org/en/8.0.x/how-to/parametrize.html#parametrize

Reply:
I took a look at ```parametrize```, but wasn't sure how to combine it with the ```@image_comparison``` decorator. I wrote the duplicate tests, but lmk if it can be made cleaner

Reply:
You can just put one decorator on top of another.  There are some examples in `test_contour.py`:
https://github.com/matplotlib/matplotlib/blob/47dc2c6c046a34341918a85130fecc41613560b3/lib/matplotlib/tests/test_contour.py#L166-L168

Reply:
Thanks, test has been fixed